app/views/finders/show.html.erb

```diff
@@ -58,6 +58,10 @@
   <% end %>
 </header>
 
+<% if finder.summary %>
+  <div class="keyline"><!-- --></div>
```

Comment:
If you wanted to do the `border-bottom` on the header, you could update Finder Frontend to use the `core_layout` from Slimmer.